VMs/metrics/ubuntu16.04/user-data.proxy

```diff
@@ -0,0 +1,108 @@
+#cloud-config
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+groups:
```

Comment:
This file has similar content to `user-data`. Is there any way to make cloud-init include the common bits to avoid the duplication?

Reply:
Sadly, not a trivial or even medium-hard way I could find. I started off with them split and `cat`ing them together, and that does not work as `cloud-init` (for some sections?) takes the last seen section - it does not auto-merge them. There are some wacky merge-rules you can set, but I think they do not apply in this instance. So, I looked for YAML merge tools - and that looked quite non-trivial as well - so I went with the split sadly. Yeah, I wasn't too impressed either.

Reply:
I guess a follow-on PR could "build" these two yaml files using some `m4`/`sed` magic maybe so we guarantee they are consistent when used?